### PR TITLE
Add payment category purpose fields to API docs

### PR DESCRIPTION
**Pull request by zachingle**
Subject to change!

Add fields to the Make a Payment endpoint needed in order to make a SuperStream compliant payment.

Screen Shot 2023-01-11 at 2 50 10 pm

### CODE REVIEW

File: source/openapi3/split.yaml [hunk 1]
```diff
@@ -2681,6 +2681,8 @@ paths:
         </ul>
 
         <aside class="notice">We strongly recommend to supply an <code>Idempotency-Key</code> header when performing this request to ensure you can safely retry the action in case of an issue. If a header value is omitted or is different to one provided previously, we will be treating a request as a new operation which may lead to duplicate payments. To understand more on how to make idempotent requests, please refer to our <a href="#idempotent-requests">Idempotent requests guide</a>.</aside>
+
+        <aside class="notice">To conduct a SuperStream compliant payment, a Payment Reference Number must be supplied to the <code>your_unique_ref</code> field and <code>PENS</code> filled in the <code>category_purpose_code</code> field </aside>
```
Comment (kieranjogrady) [inline source/openapi3/split.yaml]: A couple of nitpicks to consider:
- align our terminology when referring to payload properties
- link to ATO's SuperStream page
```suggestion
        <aside class="notice">To conduct a <a href="https://www.ato.gov.au/super/superstream/">SuperStream<\a> compliant payment, a Payment Reference Number must be supplied for the <code>your_unique_ref</code> property and <code>PENS</code> value supplied for the <code>category_purpose_code</code> property </aside>
```

**Review by chrisixion — APPROVED** [review]
Looks good!

**Comment by sharmanat0r** [comment]
Hey team, just a few notes:
- `your_unique_ref` - do we want to continue using the term `your` ?  I remember "your" being a thing a couple of years back, and we went with it but... does it still make sense? This reference will only get sent on the NPP, and never DE.
- `your_unique_ref` - this is already a secret attribute that is in use by 1 of our major merchants... are we confusing things or am I confused here 🤣 
- `category_purpose_code` - I like it as this is what it is, no-nonsense!
- do we want to mention the ISO standard? ie, do we want our merchants looking that up to include any code they want? My initial thoughts were to lock this down, ie, it's an enum that can be blank or PENS (and later maybe TAXS and SALA)

**Comment by zubin** [comment]
Marking as draft for now.

**Comment by chrisixion** [comment]
Closing draft PR

**Pull request closed without merging**